### ISSUE TITLE
Homepage stats section skips over content when tabbing with screenreader

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#4229]
**Describe the bug**
When tabbing through the homepage the network stats section has unusual behaviour when interacting with a screenreader. Focus will land on the box heading (i.e. "ETH PRICE (USD)") then focus will instantly shift to the next focusable element (30D button)

<img width="745" alt="Screenshot 2021-10-14 at 11 39 12" src="https://user-images.githubusercontent.com/62268199/137302130-e04e1b9f-7a50-4b5b-874d-745f65d62d04.png">

This is the [component](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/StatsBoxGrid.js) causing the bug.

[See a GIF of the bug here](https://i.gyazo.com/cd845c38926b94fb8eee72f096eec5c4.gif)

**To Reproduce**

1. Go to '[ethereum.org](https://ethereum.org/en/)'
2. Tab through site until you reach 'Ethereum Today - The latest network statistics section'
3. Attempt to tab onto 'ETH PRICE (USD)'

**Expected behavior**

Focus should not shift off of the tabbed element.

**Screenshots**
https://i.gyazo.com/cd845c38926b94fb8eee72f096eec5c4.gif

**Desktop (please complete the following information):**

- OS: macOS
- Browser: Brave
- Version: 1.29.81

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by minimalsm** [comment]
> do you want it to focus on "ETH PRICE (USD)" while tabbing?

No, headings/paragraphs shouldn't get focus when tabbing through the site. Need to do some triage to work out why it is highlighting these for half a second then skipping to the buttons (it is very jarring when using a screenreader)

> when I tab through the homepage the buttons of the "Ethereum Today - The latest network statistics" section do not show that they are focused/active to the user because they have a CSS styling of outline: none;

Hmm that is strange. I'm on Brave on my iMac and this works fine but I've just tested it on my laptop and it doesn't work. Yes, these should be 100% outlined for accessibility reasons. I think this line should probably be removed but we needed to use `outline: none` here and find an alternative solution.

Seems it was added when [we were adding charts to the stats box](https://github.com/ethereum/ethereum-org-website/commit/b13d28705775935c436363ceaad418eee5995227#diff-9f76ae40d9acc17724e6fc36a3253c261b9b7f087aa8e040a1afe320bad3bf63) (cc: @wackerow)

**Comment by lemiesz** [comment]
@minimalsm  

Not sure if this is your issue however ive noticed sometimes that Mac can have issues with tab navigation unless you enable some settings at the OS level. https://www.scottohara.me/blog/2014/10/03/link-tabbing-firefox-osx.html 

Ive noticed this problem with Firefox in the past.

**Comment by lemiesz** [comment]
Although i have just tried it in firefox and the tab navigation seems to be working  mostly correctly.  However with a few problems. 

<img width="1826" alt="image" src="https://user-images.githubusercontent.com/2940410/144146170-d93c2db7-611e-4854-b1cd-39e81f091dea.png"> 

The code examples section is not properly accessible to keyboard navigation. The `DeveloperPortal` button does not have a clear focus state on keyboard navigation. These time range selection buttons also don't have very clear focus states.

<img width="231" alt="image" src="https://user-images.githubusercontent.com/2940410/144146286-07bf9415-5cf0-4d77-a72d-8cd4ad697e64.png"> 

Not really what the original issues was describe but I can try to take this on if no one is on it yet.

**Comment by minimalsm** [comment]
@lemiesz thanks for offering to help 🙏 If you're up for making a PR for this that'd be awesome. Let me know and I can assign you to this issue.

**Comment by lemiesz** [comment]
Hey really sorry. Got caught up with vacation and new year, forgot I even commented on this. 


I don't have the bandwidth to work on this until possibly the end of the month. If its a high priority please assign to someone else.

**Comment by minimalsm** [comment]
@lemiesz no worries, completely understand 😀 not massively high priority if you're still interested in picking this up.

**Comment by vluna** [comment]
@minimalsm I can take a look at this one. I would just like some clarification if possible:
- Remove `outline: none;` or find a way to make sure the focus shows on the button dates *30d* and *90d*
- Make sure that after tabbing the button *Developer portal*, the focus goes to the title of the charts

If this can be confirmed or any other changes are needed, I can work on it.

**Comment by vluna** [comment]
@minimalsm just wondering if you saw my comment above.

**Comment by minimalsm** [comment]
@vluna sorry I didn't, thanks for offering to help. Yeah, we shouldn't have outline none on this (accessibility). Feel free to use your judgement here.
